autoload/keepeye.vim

```diff
@@ -51,3 +51,11 @@ function! keepeye#Start() abort
   call timer_start(g:keepeye_timer*1000, 'keepeye#CallbackWrapper')
 endfunction
 
+function! s:getHighlightGroup() abort
+  if g:keepeye_statusline_hi_group == 0
+    return ''
+  endif
+
+  return '%'.l:keepeye_statusline_hi_group.'*'
```

Comment:
`g:keepeye_statusline_hi_group` you wanted to say (instead of `l:`) ?

Reply:
Yeah. Fixed.